app/controllers/bookings_controller.rb

```diff
@@ -1,3 +1,29 @@
 class BookingsController < ApplicationController
   #TODO index, show, new, create
+
+  def index
+    @bookings = Booking.all
+  end
+
+  def show
+    @Booking = Booking.find(params[:id])
+  end
+
+  def new
+    @plant = Plant.find(params[:plant_id])
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
```

Comment:
not sure if it's me, still finding things a bit confusing, but can't see the booking_params functions. Is it because it's private and therefore not visible on github?

Reply:
I don't know. I guess you change it locally and put a new pull request. Not sure if/how you can delete this one...

Reply:
I think you can edit through here hope . put edit change the code and save it.